test/e2e/benchmarks_test.go

```diff
@@ -0,0 +1,174 @@
+//go:build benchmarks
+// +build benchmarks
+
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	. "github.com/containers/podman/v4/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var (
+	// Number of times to execute each benchmark.
+	numBenchmarkSamples = 3
+	// All benchmarks are ququed here.
+	allBenchmarks []benchmark
+)
+
+// An internal struct for queuing benchmarks.
+type benchmark struct {
+	// The name of the benchmark.
+	name string
+	// The function to execute.
+	main func()
+	// Function is run before `function`.
+	init func()
+}
+
+// Allows for customizing the benchnmark in an easy to extend way.
+type newBenchmarkOptions struct {
+	// Sets the benchmark's init function.
+	init func()
+}
+
+// Queue a new benchmark.
+func newBenchmark(name string, main func(), options *newBenchmarkOptions) {
+	bm := benchmark{name: name, main: main}
+	if options != nil {
+		bm.init = options.init
+	}
+	allBenchmarks = append(allBenchmarks, bm)
+}
+
+var _ = Describe("Podman Benchmark Suite", func() {
+	var (
+		timedir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+	)
+
+	setup := func() {
+		tempdir, err = CreateTempDirInTempDir()
```

Comment:
Looks like the error is here, `=` should be `:=` and the `err` in line 55 needs to go.

But then, if I fix that, I somehow end up with a tmpdir called `/tmp/root` which is terrifying. Now I'm trying to figure out the source of that bug, but if it's easier for you to track down, I'd appreciate getting that fixed.